src/Microsoft.AspNetCore.ResponseCompression/BrotliCompressionProviderOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NETCOREAPP2_1
```

Comment:
The public types should be NETSTANDARD, let's keep the NETCOREAPP regions as constrained as possible. You should be able to limit them to ResponseCompressionProvider.ctor and BrotliCompressionProvider.CreateStream

Reply:
But what will they do on non-netcoreapp2.1? Throw?

Reply:
CreateStream is the only place that would need to throw (NotImplementedException?). And that exception will be avoided by having ResponseCompressionProvider skip adding the provider. It would only throw if someone added it manually.

Reply:
> It would only throw if someone added it manually.

Good point 👍🏻

Reply:
I'm assuming that also means that you're fine with adding Brotli as a default provider?

Reply:
That's fine, as long as it's second on the list.